includes/gateways/stripe/includes/admin/class-give-stripe-admin-settings.php

```diff
@@ -686,95 +697,216 @@ public function stripe_modal_checkout_status( $status = 'enabled' ) {
 			$checkout_type = give_stripe_get_checkout_type();
 
 			if ( 'redirect' === $checkout_type ) {
-				 return 'give-hidden';
+				return 'give-hidden';
 			}
 
 			return '';
 		}
 
 		/**
-		 * Connect button to connect with Stripe account.
+		 * Stripe Account Manager Field.
```

Comment:
Please provide a description for this. Does this render the Stripe Account Manager Field? What does this do?

Reply:
Added description